src/main/kotlin/no/nav/familie/ks/sak/kjerne/brev/BrevPeriodeContext.kt

```diff
@@ -297,6 +298,10 @@ class BrevPeriodeContext(
                         )
                     }
 
+                val relevanteVilkårResultater = personResultater
```

Comment:
Hva betyr det at et vilkår er relevant?

Reply:
Vi finner først relevante personer for begrunnelsene.
relevanteVilkårResultater er vilkårresultatene tilhørende disse personer.

Kanskje den skal renames til vilkårResultaterTilhørendeRelevantePersoner eller noe sånt? 😛

Reply:
Det er litt vanskelig å skjønne hva det er uten å lese koden, men ser `relevantePersoner` som det hentes fra har et like generisk navn. Tror jeg synes `vilkårResultaterTilhørendeRelevantePersoner` eventuelt `vilkårResultaterForRelevantePersoner` er mer beskrivende